sample/RedHatSummit17/speakers

```diff
@@ -1,5 +1,5 @@
 [{
```

Comment:
Please refer to https://open-event-api.herokuapp.com/#speakers-speaker-get
Make changes to all the attributes.